tokio/src/runtime/task/state.rs

```diff
@@ -4,7 +4,7 @@ use std::fmt;
 use std::sync::atomic::Ordering::{AcqRel, Acquire, Release};
 use std::usize;
 
-pub(super) struct State {
+pub(crate) struct State {
```

Comment:
Why is this made `pub(crate)`? I don't see it used anywhere.

Reply:
Whoops, that fell through the cracks of an earlier revision. I'll correct it.